geoengine/types.py

```diff
@@ -640,6 +674,13 @@ def from_response(response: Dict[str, Any]) -> ClassificationMeasurement:
 
         return ClassificationMeasurement(measurement, classes)
 
+    def to_dict(self) -> Dict[str, Any]:
```

Comment:
and here

Reply:
04e3b2b514c10eeaa30770a65b6b2d4663500498